keras_cv/models/models_test.py

```diff
@@ -283,6 +283,42 @@ def test_application_variable_input_channels(self, app, last_dim):
 
         backend.clear_session()
 
+    @parameterized.parameters(*MODEL_LIST)
+    def test_rescaling(self, app, _):
+        for include_rescaling in [True, False]:
+            if "Mixer" not in app.__name__:
+                model = app(
+                    input_shape=(224, 224, 3),
+                    include_top=True,
+                    num_classes=1000,
+                    include_rescaling=include_rescaling,
+                    weights=None,
+                )
+            elif "MixerB16" in app.__name__ or "MixerL16" in app.__name__:
+                model = app(
+                    input_shape=(224, 224, 3),
+                    patch_size=(16, 16),
```

Comment:
Lets update these tests to remove hard coded checks in the test body.  

Instead, lets introduce support for "kwargs" in MODEL_LIST as an extra parameter.

So for models with no extra params you'll have:

```
    (darknet.DarkNet53, 512, {}),
```

For models that do require more info:

```
    (mlp_mixer.MLPMixerL16, 1024, {"patch_size": (16, 16))
```

Unit tests will take:

```
    def test_rescaling(self, app, args):
```

and model construction will be done:

```
model = app(
                    input_shape=(224, 224, 3),
                    include_top=True,
                    num_classes=1000,
                    include_rescaling=include_rescaling,
                    weights=None,
                    **extra_args
                )
```

Reply:
I've adopted the args model, which has cleaned up this code quite a bit. It's still imperfect due to the output_shape checks, but I think this is a big improvement.